Configurations/windows-makefile.tmpl

```diff
@@ -402,10 +416,10 @@ libclean:
 	-del /Q /F $(LIBS) libcrypto.* libssl.* ossl_static.pdb
 
 clean: libclean
-	{- join("\n\t", map { "-del /Q /F $_" } @HTMLDOCS1) -}
-	{- join("\n\t", map { "-del /Q /F $_" } @HTMLDOCS3) -}
-	{- join("\n\t", map { "-del /Q /F $_" } @HTMLDOCS5) -}
-	{- join("\n\t", map { "-del /Q /F $_" } @HTMLDOCS7) -}
+	-rmdir /Q /S $(HTMLDOCS1_BLDDIRS)
+	-rmdir /Q /S $(HTMLDOCS3_BLDDIRS)
+	-rmdir /Q /S $(HTMLDOCS5_BLDDIRS)
+	-rmdir /Q /S $(HTMLDOCS7_BLDDIRS)
```

Comment:
Something is wrong here; it seems like the variables expand to empty strings:

```diff
$ diff nmake-old.log nmake-new.log 
--- nmake-old.log	2019-12-02 12:10:28.353615678 +0100
+++ nmake-new.log	2019-12-02 12:08:56.490698028 +0100
@@ -9,28 +9,14 @@
 C:\src\openssl\providers\libcommon_static.lib konnte nicht gefunden werden
 C:\src\openssl\test\libtestutil_static.lib konnte nicht gefunden werden
 C:\src\openssl\libcrypto.* konnte nicht gefunden werden
-	del /Q /F doc\html\man1\CA.pl.html
-C:\src\openssl\doc\html\man1\CA.pl.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-asn1parse.html
-C:\src\openssl\doc\html\man1\openssl-asn1parse.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-ca.html
-C:\src\openssl\doc\html\man1\openssl-ca.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-ciphers.html
-C:\src\openssl\doc\html\man1\openssl-ciphers.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-cmds.html
-C:\src\openssl\doc\html\man1\openssl-cmds.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-cms.html
-C:\src\openssl\doc\html\man1\openssl-cms.html konnte nicht gefunden werden
-	del /Q /F doc\html\man1\openssl-crl.html
-
-...
-
-
-C:\src\openssl\doc\html\man7\proxy-certificates.html konnte nicht gefunden werden
-	del /Q /F doc\html\man7\ssl.html
-C:\src\openssl\doc\html\man7\ssl.html konnte nicht gefunden werden
-	del /Q /F doc\html\man7\x509.html
-C:\src\openssl\doc\html\man7\x509.html konnte nicht gefunden werden
+	rmdir /Q /S 
+Syntaxfehler.
+	rmdir /Q /S 
+Syntaxfehler.
+	rmdir /Q /S 
+Syntaxfehler.
+	rmdir /Q /S 
+Syntaxfehler.
 	del /Q /F apps\openssl.exe
 C:\src\openssl\apps\openssl.exe konnte nicht gefunden werden
 	del /Q /F fuzz\asn1-test.exe
```

Reply:
I just observed the exact same thing.  Debugging reveals that the build.info data is generated correctly, but still, nothing ends up in configdata.pm...

Reply:
Finally figured it out.  Stoopid error, and as usual, they take time to actually see.

Reply:
I cleaned a full build and watched the performance. Your fix works and performance is much better than at the beginning of the pr. One more possible optimization would be to delete the test executables using a wild card.
```
 del /Q /F test\*.exe
``` 
Deleting every single one in a separate command is very slow. But you don't have to do it if you don't want to.